docs/03-pages/01-building-your-application/06-configuring/13-debugging.mdx

```diff
@@ -33,7 +33,7 @@ Create a file named `.vscode/launch.json` at the root of your project with the f
       "request": "launch",
       "command": "npm run dev",
       "serverReadyAction": {
-        "pattern": "started server on .+, url: (https?://.+)",
+        "pattern": "Local:.+(https?:\/\/.+)",
```

Comment:
```suggestion
        "pattern": "- Local:.+(https?:\/\/.+)",
```

Perhaps we should look for the `-` prefix like we do here:

https://github.com/vercel/next.js/blob/563019f79cb77fae8ce809160ecea142bb16c1bd/test/lib/next-modes/next-dev.ts#L98

Reply:
Either works, but I added it for consistency